Q24ECU/core/src/interfaces/interface_uart.c

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file interface_uart.c
+ * @author Jacob Chisholm (Jchisholm204.github.io)
+ * @brief 
+ * @version 0.1
+ * @date 2023-11-18
+ * 
+ * @copyright Copyright (c) 2023
+ * 
+ */
+
+#include "interfaces/interface_uart.h"
+#include "FreeRTOS.h"
+#include "task.h"
+#include "taskHandlers.h"
+#include "nvicConfig.h"
+
+
+uart_t debug;
+
+void uart_setup(){
+    // Enable the debug usart and setup its IQR handler
+    uart_send_init(&debug, UART_DEBUG, 250000);
```

Comment:
Will we ever use anything other than the `debug` USART? If so should this function be changed to accept any USART in the params of the function?

Reply:
Yes, I just added the debug port as an example for how USART should be implemented. I will change to make the code more clear